### ISSUE TITLE
Invalid config HA 2023.11 - unable to prepare setup for platform thermiagenesis.climate

### ISSUE THREAD

**Issue by kejXero** [CJNE/thermiagenesis#239]
Thank you for your great work on this integration. My issue started when I updated my Home assistant to HA2023.11. After this update the climate.heat entity disappeared and the possibility to set the heat target also disappeared.

I tried without success, a reinstall if your integration, with a subsequently re-activation of used entities. Some "feed back info" changed from ON to PLUGGED IN.

Log copy from HA:
Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature').
November 20, 2023 at 21:32:51 – (ERROR) setup.py

Googling tells me that that that module was deprecated a year ago, so I’m guessing it was finally removed in 2023.11.

My Thermia as an Thermia Calibra 12

## Version of the custom_component
Version 0.0.10

## Configuration

```yaml
Logger: homeassistant.setup
Source: setup.py:359
First occurred: 20:08:44 (1 occurrences)
Last logged: 20:08:44

Unable to prepare setup for platform 
```
HA 2023.11.3

## Describe the bug

After update to HA 2023.11 the comfort wheel disappeared (climate.heat entity).  I tried to uninstall the integration and subsequently reinstalled it with out any succes.

## Debug log

```text

Add your logs here.
Logger: homeassistant.setup
Source: setup.py:359
First occurred: 20:08:44 (1 occurrences)
Last logged: 20:08:44

Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature').
```

**Comment by kejXero** [comment]
Thank you for your great work on this integration. My issue started when I updated my Home assistant to HA2023.11. After this update the climate.heat entity disappeared and the possibility to set the heat target also disappeared.

I tried without success, a reinstall if your integration, with a subsequently re-activation of used entities. Some "feed back info" changed from ON to PLUGGED IN.

Log copy from HA:
Unable to prepare setup for platform thermiagenesis.climate: Platform not found (No module named 'homeassistant.util.temperature').
November 20, 2023 at 21:32:51 – (ERROR) setup.py

Googling tells me that that that module was deprecated a year ago, so I’m guessing it was finally removed in 2023.11.

My Thermia as an Thermia Calibra 12
